### ISSUE TITLE
Urgent Fixes For We Whom Use VoiceOver On Mac OS

### ISSUE THREAD

**Issue by Rogue-Git-Dev** [transmission/transmission#1024]
A myriad of implementations must be made for accessibility. I will mention the most important of them but others exist I am sure.

* Most front-facing is the unlabelled inaccessible table. It seems to have six columns. From left to right, VoiceOver reports ‘image, blank, image, blank, image, blank’. This tells me nothing, and is one of the most inaccessible aspects of Transmission on Mac OS.
* Inspector invoked by ⌘i contains five unlabelled buttons and other unlabelled elements. VoiceOver says the word ‘button’ for each button without label. Moreover, VoiceOver fails to convey which is selected.
* An unlabelled button exists after the  Data File information on the General Info tab.
* On the Activity tab, two tick boxes labelled Progress and Available seemingly change nothing when either are ticked. Are they necessary? VoiceOver reports the same data regardless.
* Another inaccessible and unlabelled table exists on the Files tab. The tick boxes have no label, and nothing is spoken except for ‘expanded’ and ‘collapsed’. This table is equally useless as that on the main screen.

**Comment by Rogue-Git-Dev** [comment]
Although this issue is original, my duplicate issue is more verbose and thus this must be closed.
